pkg/core/lifecycle.go

```diff
@@ -80,11 +80,17 @@ func (pl *Pipeline) Start(ctx context.Context) (err error) {
 		StartTime:   startTime,
 		Status:      Running,
 	}
+	mode := "discovery"
 	if pl.Cfg.DiscoverMode {
 		taskPayload.Type = DiscoveryTask
+	} else if pl.Cfg.FlakyMode {
+		mode = "flaky"
+		taskPayload.Type = FlakyTask
 	} else {
+		mode = "execution"
 		taskPayload.Type = ExecutionTask
 	}
+	pl.Logger.Infof("Running nucleus in %s mode", mode)
```

Comment:
instead of `mode` (a new variable), can we reuse `taskPayload.Type` for the info log?

Reply:
done